IDisposableAnalyzers/IDisposableAnalyzers.csproj

```diff
@@ -66,7 +66,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="tools\*" Pack="true" PackagePath="" />
+    <None Update="tools\*;build\*" Pack="true" PackagePath="" />
+    <Compile Remove="build\*.cs" />
+    <None Include="build\*.cs" Pack="true" PackagePath="build" />
```

Comment:
Isn't there duplicated files with the `None Update build\*` above?

Reply:
@jeremyVignelles I don't think so. Only if there were `.cs` files added as `<None>` items which there shouldn't be.